Update READMEs to add details on building playground and katas (#1402)
microsoft/qsharp: README.md

### Playground

> `playground` module builds the following 3 things locally:

Reviewer: ```suggestion
`playground` module builds the following components locally:
```

### Playground

`playground` module builds the following components locally:

- katas
- samples
- documentation

> To see instructions about building playground , refer to [Building PlayGround Locally](./playground/README.md#building-playground-locally).

Reviewer: ```suggestion
To see instructions about building playground , refer to [Building Playground Locally](./playground/README.md#building-playground-locally).
```

> `playground` module builds the following components locally:
> 
> - katas
> - samples
> - documentation
> 
> To see instructions about building playground , refer to [Building Playground Locally](./playground/README.md#building-playground-locally).

Reviewer 2: ```suggestion
The `playground` is a small website that loads the Q# editor, compiler, samples, katas, and documentation for the standard library. It's a way to manually validate any changes you make to these components.

To see instructions for building the playground, refer to [Building the Playground Locally](./playground/README.md#building-the-playground-locally).
```

A bit more technically accurate. Hopefully still informative?
